addons/web_hierarchy/static/src/hierarchy_card.scss

```diff
@@ -50,15 +50,6 @@
             }
         }
 
-        .o_hierarchy_node_body {
```

Comment:
this class is still used above, should we keep it or remove it above ?

Reply:
i will remove every occurence of this class seems useless :/